lib/project-view.coffee

```diff
@@ -56,16 +58,25 @@ class ProjectView extends SymbolsView
       @reloadTags = @tags.length is 0
       @setItems(@tags)
 
-    @watchTagsFile()
+    @watchTagsFiles()
 
-  watchTagsFile: ->
-    @unwatchTagsFile()
+  watchTagsFiles: ->
+    @unwatchTagsFiles()
 
-    if tagsFilePath = TagReader.getTagsFile()
-      @tagsFile = new File(tagsFilePath)
-      @tagsFile.on 'moved removed contents-changed', =>
```

Comment:
While reviewing the diff I saw this code (which was old) and realized we hadn't been marking deprecations for these kinds of `File::on` calls.

I updated the PR to the new API.